src/Adapter/MSTest.CoreAdapter/ObjectModel/UnitTestElement.cs

```diff
@@ -72,6 +72,11 @@ public UnitTestElement(TestMethod testMethod)
         /// </summary>
         public KeyValuePair<string, string>[] DeploymentItems { get; set; }
 
+        /// <summary>
+        /// Gets or sets the DisplayName
+        /// </summary>
+        public string DisplayName { get; set; }
```

Comment:
@jayaranigarg , @singhsarab : would this cause any compat issues?

Reply:
Can we validate if this works in a solution that contains a project that references this version of the adapter and an older version of the adapter? I'd expect tests from both the projects to run.

Reply:
( Describing what I've done to make sure I got correctly what you meant. 🙂 )

Have 4 projects 2 on Core, and 2 on Framework. In all projects I updated the `MSTest.TestFramework` package to the nuget package that was built locally (`v99.99.99-dev`), this allows the display name to be specified. In one project for each platform I updated also the `MSTest.TestAdapter`.

In all projects I have `TestMethod1` decorated with `[TestMethod("FriendlyName")]`.

In the results I see that that using older adapter on Framework correctly ignores the DisplayName, but on any other combination it does not work, not even when the packages are both updated.

Looks like the display name is used even where the actual method name should be used. Will investigate. 

![image](https://user-images.githubusercontent.com/5735905/70994859-6307dc00-20cf-11ea-948d-743dbdf0ec49.png)